support/stunnel-instrumented-with-springboard/stunnel.conf.template

```diff
@@ -9,6 +9,7 @@ foreground=yes
 accept=${STUNNEL_ACCEPT_PORT}
 client=${STUNNEL_CLIENT_MODE}
 connect=${STUNNEL_CONNECT}
+delay=yes
```

Comment:
The documentation doesn't mention it is done for every connection explicitly. Is the container restarted for every connection in which case this can be inferred?

Reply:
True, the doc is ambiguous and appears to imply that the resolution is merely delayed once. But inspection of the code and observation of the runtime behavior confirms that the resolution is performed every time.